pikaday.js

```diff
@@ -425,7 +427,13 @@
                 opts.i18n.months[i] + '</option>');
         }
 
-        monthHtml = '<div class="pika-label">' + opts.i18n.months[month] + '<select class="pika-select pika-select-month" tabindex="-1">' + arr.join('') + '</select></div>';
+        monthInnerHtml = opts.i18n.months[month] + '<select class="pika-select pika-select-month" tabindex="-1">' + arr.join('');
+
+        if (instance.el.querySelector('div#' + randId + ' .pika-label-month')) {
```

Comment:
we can probably assign this whole thing into a var

```
var pikaLabelMonth = instance.el.querySelector('div#' + randId + ' .pika-label-month')
```

and then use it 

```
if (pikaLabelMonth) {
   pikaLabelMonth.innerHTML = monthInnerHtml;
}
```

wdyt. this way we don't need to access the DOM twice